platform/core/src/utils/isDisplaySetReconstructable.js

```diff
@@ -31,37 +33,43 @@ export default function isDisplaySetReconstructable(instances) {
 
 function processMultiframe(instance) {
   //TODO: deal with multriframe checks! return false for now as can't reconstruct.
-  return { value: false };
+  const warningIssues = [reconstructionIssues.MULTIFRAMES];
+  return { value: false, warningIssues };
```

Comment:
It's not clear that the processMultiframe will return an object with two values. Let's add jsDoc to this one and processSingleFrame.